packages/components/src/components/button/button.tsx

```diff
@@ -94,7 +94,8 @@ export class Button {
           onClick={this.handleClick.bind(this)}
           rel={this.target === '_blank' ? 'noopener noreferrer' : undefined}
           onFocus={(event) => this.handleFocus(event)}
-
+          aria-disabled={this.disabled}
+          aria-labelledby="label"
```

Comment:
again here id propose to have a aria property that the user can set, to give some context